sdk/core/System.Memory.Data/src/ReadOnlyMemoryStream.cs

```diff
@@ -8,30 +8,44 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Azure.Core
+namespace System
```

Comment:
Nit: this really should be System.IO. But it's internal, so we can always change it when we move it to runtime.